draft-ietf-cbor-7049bis.md

```diff
@@ -2119,6 +2121,18 @@ to be clearly documented in the decoder.  Beyond the validity at the CBOR level,
 application also needs to ascertain that the input is in alignment
 with the application protocol that is serialized in CBOR.
 
+The input check itself may consume resources.  This is usually linear
+in the size of the input, which means that an attacker has to spend
+resources that are commensurate to the resources spent by the defender
+on input validation.  Processing for arbitrary-precision numbers may
+exceed linear effort.  Also, some hash-table implementations that are
+used by decoders to build in-memory representations of maps can be
+attacked to spend quadratic effort, unless a secret key is employed
```

Comment:
The summary of "use a secret key" isn't sufficient by itself, although the reference to [SIPHASH] helps: http://emboss.github.io/blog/2012/12/14/breaking-murmur-hash-flooding-dos-reloaded/#targetText=Here%20come%20the%20cryptographers

Maybe use "unless a secure hash with a secret key is employed"?